docs/_apps/distil-whisper-wrapper/v1.0/index.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: posts
+classes: wide
+title: "Distil Whisper Wrapper (v1.0)"
+date: 2024-07-17T22:27:16+00:00
+---
+## About this version
+
+- Submitter: [1192119703jzx](https://github.com/1192119703jzx)
+- Submission Time: 2024-07-17T22:27:16+00:00
+- Prebuilt Container Image: [ghcr.io/clamsproject/app-distil-whisper-wrapper:v1.0](https://github.com/clamsproject/app-distil-whisper-wrapper/pkgs/container/app-distil-whisper-wrapper/v1.0)
+- Release Notes
+
+    (no notes provided by the developer)
+
+## About this app (See raw [metadata.json](metadata.json))
+
+**The wrapper of Distil-Whisper, avaliable models: distil-large-v3, distil-large-v2, distil-medium.en, distil-small.en. The default model is distil-small.en.**
+
+- App ID: [http://apps.clams.ai/distil-whisper-wrapper/v1.0](http://apps.clams.ai/distil-whisper-wrapper/v1.0)
+- App License: Apache 2.0
+- Source Repository: [https://github.com/clamsproject/app-distil-whisper-wrapper](https://github.com/clamsproject/app-distil-whisper-wrapper) ([source tree of the submitted version](https://github.com/clamsproject/app-distil-whisper-wrapper/tree/v1.0))
+- Analyzer Version: 1.0
+- Analyzer License: MIT
+
+
+#### Inputs
+(**Note**: "*" as a property value means that the property is required but can be any value.)
+
+One of the following is required: [
+- [http://mmif.clams.ai/vocabulary/AudioDocument/v1](http://mmif.clams.ai/vocabulary/AudioDocument/v1) (required)
+(of any properties)
+
+- [http://mmif.clams.ai/vocabulary/VideoDocument/v1](http://mmif.clams.ai/vocabulary/VideoDocument/v1) (required)
+(of any properties)
+
+
+
+]
+
+
+#### Configurable Parameters
+(**Note**: _Multivalued_ means the parameter can have one or more values.)
+
+- `modelSize`: optional, defaults to `distil-small.en`
+
+    - Type: string
+    - Multivalued: False
+    - Choices: `distil-large-v3`, `distil-large-v2`, `distil-medium.en`, **_`distil-small.en`_**, `small`, `s`, `medium`, `m`, `large-v2`, `l2`, `large-v3`, `l3`
+
+
+    > The size of the model to use. There are four size of model to use distil-large-v3, distil-large-v2, distil-medium.en, distil-small.en. The default model is distil-medium.en.)
```

Comment:
A little thing, but I think it'd helpful if we provide what those "aliases" are in the parameter description.